src/app/utils/envHelper.ts

```
@@ -0,0 +1,77 @@
+  return obj[prop];
+};
+
+// Get if we're in development mode
+export const isDev = (): boolean => {
+  // Only try to use import.meta in browser context
+  if (isBrowser()) {
+    try {
+      // We need to use eval to avoid TypeScript errors during compilation
+      // This code will only run in the browser context where import.meta is available
+
+      const meta = eval('import.meta');
```

Biome (1.9.4)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)